package-lock.json

```diff
@@ -33782,28 +33782,28 @@
           "dependencies": {
             "abbrev": {
               "version": "1.1.1",
-              "resolved": false,
```

Comment:
Please revert these changes in the package.lock, this will otherwise be included again in another PR